registration/pom.xml

```diff
@@ -106,10 +106,14 @@
 		<jackson.version>2.15.4</jackson.version>
 		<jackson.mapper.asl.version>1.9.7</jackson.mapper.asl.version>
 		<mosip.registration.client.version>1.3.0-SNAPSHOT</mosip.registration.client.version>
-		<mosip.core.kernel.version>1.3.0-SNAPSHOT</mosip.core.kernel.version>
+		<mosip.core.kernel.version>1.3.0-beta.1</mosip.core.kernel.version>
 		<mosip.biometric.util.version>1.3.0-SNAPSHOT</mosip.biometric.util.version>
 		<mosip.commons.packet.manager.version>1.3.0-SNAPSHOT</mosip.commons.packet.manager.version>
 		<mosip.kernel.virusscanner.version>1.1.3</mosip.kernel.virusscanner.version>
+
+        <!--        -->
+        <mosip.kernel-auditmanager-api.version>1.3.0-SNAPSHOT</mosip.kernel-auditmanager-api.version>
+        <mosip.bio-utils.version>1.3.0-SNAPSHOT</mosip.bio-utils.version>

```

Comment:
Can't we use <mosip.biometric.util.version> property instead of creating new one.

Reply:
yes. changed to mosip.biometric.util.version